MetricsUploader/MetricsUploaderWindows.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/memory/memory.h>
+
+#include "MetricsUploader/MetricsUploader.h"
+#include "OrbitBase/Logging.h"
+
+namespace orbit_metrics_uploader {
+
+constexpr const char* kSendLogEventFunctionName = "SendOrbitLogEvent";
+constexpr const char* kShutdownUploaderClientConnName = "ShutdownUploaderClientConn";
+constexpr const char* kStartUploaderClientFunctionName = "StartUploaderClient";
+
+MetricsUploader::~MetricsUploader() {
+  absl::MutexLock lock(&send_log_event_mutex_);
+  send_log_event_addr_ = nullptr;
```

Comment:
Setting this to `nullptr` is not necessary, since the object gets destructed anyway.

Reply:
Right, thank you